process/smartContract/scQueryService.go

```diff
@@ -114,9 +134,26 @@ func (service *SCQueryService) executeScCall(query *process.SCQuery, gasPrice ui
 		}
 	}
 
+	if query.SameScState {
+		err := service.checkForRootHashChanges(rootHashBeforeExecution)
```

Comment:
No need for a new `err` declaration, there's one on line 114.

Reply:
done